app/components/ProjectTemplate.tsx

```diff
@@ -0,0 +1,151 @@
+import { FC } from "react";
+
+import type { CodeSeeMapMetadata, GitHubData, Project } from "~/types";
+import ProjectAvatar from "~/components/ProjectAvatar";
+import RepoLinks from "~/components/RepoLinks";
+import RepoStats from "~/components/RepoStats";
+import ProjectTabs from "~/components/ProjectTabs";
+import Tag from "~/components/Tag";
+import LearnSection from "~/components/markdown/LearnSection";
+import AnchorHeader from "~/components/markdown/AnchorHeader";
+import ContributionOverview from "~/components/markdown/ContributionOverview";
+import HelpWanted from "~/components/markdown/HelpWanted";
+import HacktoberfestIssues from "~/components/markdown/HacktoberfestIssues";
+import FeaturedCodeSeeMap from "~/components/markdown/FeaturedCodeSeeMap";
+import Maps from "~/components/Maps";
+
+type Props = {
+  project: Project;
+  githubData?: GitHubData;
+  featuredMapMetadata?: CodeSeeMapMetadata;
+};
+
+/**
+ * Renders a project. This is broken out from the project page so we can reuse
+ * it as a preview.
+ */
+const ProjectTemplate: FC<Props> = ({
```

Comment:
To build the `ProjectTemplate`, I moved all the rendering code from the regular "project" page over into this component. This means that we get the exact same UI wherever we use this.